docs/_docs/04_community/client_factory.md

```diff
@@ -0,0 +1,176 @@
+# Azure SDK Client Factory
```

Comment:
In order for this to show up in the docs site, it needs a front matter. See the other `.md` files in the same folder for an example!

Reply:
Added and resolved.